laika/downloader.py

```diff
@@ -376,16 +373,11 @@ def download_prediction_orbits_russia_src(gps_time, cache_dir):
   # Example: Stark_1D_22060100.sp3
   folder_and_file_names = [(folder_path, file_prefix + f"{h:02}.sp3") for h in reversed(current_day)] + \
                           [(folder_path_prev, file_prefix_prev + f"{h:02}.sp3") for h in reversed(prev_day)]
-  file = download_and_cache_file_return_first_success(url_bases, folder_and_file_names, cache_subdir)
-  if file is None: # todo remove this when gnss-alt repo is fixed
```

Comment:
Is this fixed? Maybe something for a separate PR in case this needs to be reverted.